### DISCUSSION TITLE
Migration from Jekyll to Eleventy

### DISCUSSION THREAD

**Discussion by deepfriedbrain** [mmistakes/minimal-mistakes#3807] [Q&A]
Recently I came across Eleventy and want to try migrating to it from Jekyll for my static sites.

But I love the minimal-mistakes theme and want to keep using it for my blog. Has anyone migrated to Eleventy and using this theme? What would it take to migrate this theme? I''m not a Ruby person at all, but I'm somewhat comfortable with Javascript.

**Comment by antgel** [comment]
We're in the same boat; I was going to ask the same thing but I did a search and found this thread, so I'll piggy-back on it. I found https://github.com/quinnanya/eleven-mistakes and asked about the status https://github.com/quinnanya/eleven-mistakes/issues/1 because it seems a little unclear at present. It looks like something hacked together for single-use rather than something production-ready / generic, but ICBW.

FWIW, I don't think one would have to be a "Ruby person" to attempt this, because most of the power is in the templates, CSS etc. I could chip away at the edges but I don't think I have the expertise to lead something like this. It would be interesting to know what @mmistakes thinks, being responsible for creating this in the first place. Definitely a fun project to hack away at.

Edit: What I've done so far is to copy my content into a new Eleventy site based on https://github.com/11ty/eleventy-base-blog just to get the basics down and see what sort of basic edits need to be made before I see anything at all. I'm now starting to copy layouts across, and that's where I assume I'll open a can of worms. I'm *not* suggesting I'll get anywhere big time with this, but if I do I'll make a new theme repository (no gem / npm themes with Eleventy) to share progress.
